changelogs/fragments/2194-zos_mvs_raw-Supporting-for-volume-definition-in-zos_mvs_raw.yml

```
@@ -0,0 +1,7 @@
+minor_changes:
+   - zos_mvs_raw - Added support for volume definition in zos_mvs_raw module to allow specifying unit and disposition parameters.
```

As this will be in the release notes we can simplify this for users
```suggestion
   - zos_mvs_raw - Adds support for volume data definition.
```

---

resolved